sos/report/plugins/logs.py

```diff
@@ -54,6 +54,7 @@ def setup(self):
         ])
 
         self.add_cmd_output("journalctl --disk-usage")
+        self.add_cmd_output("journalctl -b 0 -t nvidia-persistenced")
```

Comment:
Wrong place for this, put this in the `nvidia` plugin as well, please. There's also an `add_journal()` method available which is the preferred way to get actual journals, e.g.:

```python
self.add_journal(boot=0, identifier='nvidia-persistenced')
```